app.js

```diff
@@ -3,56 +3,23 @@
 //An array for use when creating the sales projections per hour
 var storeHours = ['6am', '7am', '8am', '9am', '10am', '11am', '12pm', '1pm', '2pm', '3pm', '4pm', '5pm', '6pm', '7pm', '8pm'];
 
-//This is where all of the stores are stored as objects
-var firstAndPike = {
-  name: 'firstAndPike',
-  minCustomers:23,
-  maxCustomers:65,
-  avgCookies:6.3,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (65 - 23)) + 23;
-  },
-  cookiesPerHour: []
-};
-var seatacAirport = {
-  name: 'seatacAirport',
-  minCustomers:3,
-  maxCustomers:24,
-  avgCookies:1.2,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (24 - 3)) + 3;
-  },
-  cookiesPerHour: []
-};
-var seattleCenter = {
-  name: 'seattleCenter',
-  minCustomers:11,
-  maxCustomers:38,
-  avgCookies:3.7,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (38 - 11)) + 11;
-  },
-  cookiesPerHour: []
-};
-var capitolHill = {
-  name: 'capitolHill',
-  minCustomers:20,
-  maxCustomers:38,
-  avgCookies:2.3,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (38 - 20)) + 20;
-  },
-  cookiesPerHour: []
-};
-var alki = {
-  name: 'alki',
-  minCustomers:2,
-  maxCustomers:16,
-  avgCookies:4.6,
-  randomCustomers: function(){
-    return Math.floor(Math.random() * (16 - 2)) + 2;
-  },
-  cookiesPerHour: []
+function CookieStore(name, minCustomers, maxCustomers, avgCookies){
+  this.name = name;
+  this.minCustomers = minCustomers;
+  this.maxCustomers = maxCustomers;
+  this.avgCookies = avgCookies;
+  this.cookiesPerHour = [];
+  this.range = maxCustomers - minCustomers;
+}
+
+var firstAndPike = new CookieStore('First and Pike', 23, 65, 6.3);
```

Comment:
Move these lines down below where you attach methods to the prototype, or better yet, down near the bottom of the file. That way they won't disrupt the definition of CookieStore.

Also, consider putting all these stores into an array.